src/shared/database/setup.ts

```diff
@@ -179,15 +179,21 @@ async function setupNetworksTable(): Promise<void> {
       table.string('unls')
       table.primary(['entry'])
     })
+  }
+  const networksCount = await query('networks').countDistinct('id')
+  const networksIds = await query('networks').pluck('id')
+  if (networksCount[0].count < networks.length) {
```

Comment:
What's the objective of this if statement?

Reply:
Good catch. Yea we dont need it after combined.